content/docs/blog/index.md

```diff
@@ -17,4 +17,4 @@ This post shows you how to use different secret service providers using the new
 
 ## Banzai Cloud blog posts
 
-The developers of Banzai Cloud wrote a lot about Bank-Vaults. Their posts are now available on the [Cisco Tech Blog](https://techblog.cisco.com/tags/bank-vaults).
+The developers of Banzai Cloud wrote a lot about Bank-Vaults. Their posts are now available on the [Outshift by Cisco Blog](https://outshift.cisco.com/blog/topic/bank-vaults-).
```

Comment:
Should the other link be also updated?

Reply:
I think no, we can update it when they have fixed the blog posts on the site.